reference/media-types.md

```diff
@@ -387,6 +387,42 @@ responds to `foobar`.
 Praxis media types can declare attributes that are ordered collections of other
 media types or other Attributor types.
 
+### Praxis::Collection
+
+Praxis includes a helper, `Praxis::Collection.of(media_type)`, for working with  collections of a given media type. (*Note*: Previously, `Praxis::Collection.of` supported types that did not derive from `MediaType`. This usage has been deprecated and will be removed in a future version. Please use `Attributor::Collection.of` directly instead.)
+
+This defines a `<media_type>::Collection` type that is an `Attributor::Collection.of(<media_type>)`, that also includes  the `MediaTypeCommon` module. By default, Praxis will take the identifier of `<media_type>` and append a "collection=true" suffix to it.
+
+For example, your blog media type might have an attribute that is a collection of posts:
+
+{% highlight ruby %}
+class Blog < Praxis::MediaType
+  attributes do
+    attribute :id, Integer
+    attribute :posts, Praxis::Collection.of(Post)
+  end
+end
+{% endhighlight %}
+
+Praxis will look for a `Post::Collection` class to use for the `posts` attribute above, or create one if it does not exist. In the common case, the Praxis-defined `<media_type>::Collection` will be sufficient for any collection of media type instances. You can, however, define your own explicitly and Praxis will use that instead.
```

Comment:
I'd also add:
As stated above, the created inner Collection class will simply be an Attributor::Collection that wraps members of a given MediaType, and that has an identifier that matches the member's identifier, with an added "collection=true" suffix. So in the above case, the `Posts::Collection` will have an identifier of `application/vnd.acme.post;collection=true`.